blocks/price/price.css

```diff
@@ -0,0 +1,22 @@
+.price{
+  padding: 160px 130px 100px;
+  background-color: #f2f2f2;
+  background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 123"><polygon points="0,0 1200,0 0,95" fill="white"/></svg>');
```

Comment:
That's great. Just for a side note that you can also use `clip-path` css property to create such effect. Here is a tool - [https://bennettfeely.com/clippy/](https://bennettfeely.com/clippy/) for the same.

Reply:
@5hraddha Thanks. I'll be in the loop on that.